src/hookEvent.h

```diff
@@ -0,0 +1,32 @@
+// Copyright 2015, Christopher J. Foster and the other displaz contributors.
+// Use of this code is governed by the BSD-style license found in LICENSE.txt
+
+#ifndef DISPLAZ_HOOKEVENT_H_INCLUDED
+#define DISPLAZ_HOOKEVENT_H_INCLUDED
+
+#include <QByteArray>
+#include <QKeySequence>
+#include <QShortcut>
+
+/// Class for registering custom hook events. Implemented via QShortCut,
+/// i.e. for QKeySequences only.
+class hookEvent : public QObject
```

Comment:
Naming: this should be `HookEvent` in keeping with naming of the other classes.  For new classes, I'm also trying to follow the convention that the file is named exactly after the class, so the files should be `HookEvent.{h,cpp}`